.github/workflows/ci.yml

```
@@ -46,6 +51,7 @@ jobs:
       uses: actions/setup-node@39370e3970a6d050c480ffad4ff0ed4d3fdee5af # 4.1.0
       with:
         node-version: ${{ matrix.node.version }}
+        env: NODE_OPTIONS="${{ matrix.node.options }}"
         cache: 'npm'
     - run: npm ci
     - run: npm run test:unit
```

```suggestion
    - run: node --run test:unit
```

now we can use --run because we don't support v20